repo/meta/index.json

```diff
@@ -101,7 +101,7 @@
       ],
       "versions":{
         "0.1.0":"0",
-        "0.1.1":"1"
+        "0.1.1":"2"
```

Comment:
It looks like the tag was bumped for the Docker image, but it was missed in the index.